packages/core/platform/bundles/paas-full.yaml

```diff
@@ -155,6 +155,12 @@ releases:
   privileged: true
   dependsOn: [piraeus-operator,cilium,kubeovn,cert-manager]
 
+- name: snapshot-controller
```

Comment:
Let's also added it to `distro-full` bundle

Reply:
Remove from paas-full and add only to distro-full?

Reply:
Didn`t noticed `also` ))

Reply:
No we need it in both `distro-full` and `paas-full` bundles as they ship storage